number_trivia_app/android/app/build.gradle

```diff
@@ -0,0 +1,71 @@
+def localProperties = new Properties()
```

Comment:
Please consider removing platforms that are not being used. Android and iOS might be the ones that could be kept